repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -4628,4 +4679,14 @@ private void cacheDifferentialEntity(AtlasVertex ev, Set<String> existingValues)
         requestContext.cacheDifferentialEntity(diffEntity);
     }
 
+    private void cacheDifferentialMeshEntity(AtlasVertex ev, Set<String> existingValues) {
+        AtlasEntity diffEntity = new AtlasEntity(ev.getProperty(TYPE_NAME_PROPERTY_KEY, String.class));
+        diffEntity.setGuid(ev.getProperty(GUID_PROPERTY_KEY, String.class));
+        diffEntity.setAttribute(DOMAIN_GUIDS_ATTR, existingValues);
+        diffEntity.setUpdatedBy(ev.getProperty(MODIFIED_BY_KEY, String.class));
+        diffEntity.setUpdateTime(new Date(RequestContext.get().getRequestTime()));
+
+        RequestContext requestContext = RequestContext.get();
+        requestContext.cacheDifferentialEntity(diffEntity);
+    }
```

Comment:
Please create a new helper method which will do the common part in this as well as previous mentod `cacheDifferentialEntity` 
New method will accept diffEntity & ev, set common attributes on diffEntity & add it into requestContext